enterprise/frontend/src/embedding-sdk/test/__support__/ui.tsx

```diff
@@ -73,10 +75,20 @@ export function renderWithSDKProviders(
     );
   };
 
-  const utils = render(ui, {
-    wrapper,
-    ...options,
-  });
+  const renderFn = () =>
+    render(ui, {
+      wrapper,
+      ...options,
+    });
+  let utils: any;
+
+  if (wrapRenderInAct) {
+    await act(async () => {
+      utils = renderFn();
```

Comment:
Could you confirm that `utils` is set in the return in this `wrapRenderInAct` case? I feel like `utils` might still be unset by the time the function returns